test_conformance/images/kernel_read_write/test_write_3D.cpp

```diff
@@ -445,6 +446,17 @@ int test_write_image_3D( cl_device_id device, cl_context context, cl_command_que
                         }
                         else
                         {
+                            // mask off the top bit (bit 15) if the image
+                            // format is (CL_UNORM_SHORT_555, CL_RGB).
+                            // (Note: OpenCL says: the top bit is undefined
+                            // meaning it can be either 0 or 1.)
+                            if (imageInfo->format->image_channel_data_type
+                                == CL_UNORM_SHORT_555)
+                            {
+                                cl_ushort *temp = (cl_ushort *)resultPtr;
+                                temp[0] &= 0xefff;
+                            }
```

Comment:
Any reason this didn't call `filter_undefined_bits()`?

Reply:
MIssed it while i was refactoring :). thanks for the catch, I'll update it

Reply:
Should be fixed now (apologies for the force push on the branch(I had to rework the original commit message)